ankerctl.py

```diff
@@ -440,20 +444,29 @@ def config_show(env):
             return
 
         log.info("Account:")
-        print(f"    user_id:    {cfg.account.user_id[:20]}...<REDACTED>")
-        print(f"    auth_token: {cfg.account.auth_token[:20]}...<REDACTED>")
+        print(f"    user_id:    {cfg.account.user_id[:10]}...<REDACTED>")
+        print(f"    auth_token: {cfg.account.auth_token[:10]}...<REDACTED>")
         print(f"    email:      {cfg.account.email}")
         print(f"    region:     {cfg.account.region.upper()}")
         print()
 
         log.info("Printers:")
-        for p in cfg.printers:
+        # Sort the list of printers by printer.id
+        cfg.printers.sort(key=lambda p: p.id)
```

Comment:
I just realized that we don't need to sort here.

When we import the printer list from the http api (in `cli/config.py`), we do get the list back unsorted, and so we need to sort that.

But this value comes from our own config file, which is stored as a list of dictionaries, and as such the order is preserved.

Let's remove this sort, it's a NOP :)